core/src/main/cpp/MslError.h

```diff
@@ -310,6 +310,11 @@ class MslError
      */
     inline bool operator!=(const MslError& rhs) const { return !(*this == rhs); }
 
+    // operator<< for easier use with output
```

Comment:
As we discussed, this should be a non-member friend. You can see how to do this [here](http://www.prenhall.com/divisions/esm/app/kafura/secure/chapter8/html/8.3_operator_overloading.htm)